src/server/saved_objects/serialization/serialization.test.ts

```diff
@@ -130,17 +132,46 @@ describe('saved object conversion', () => {
       expect(actual).not.toHaveProperty('version');
     });
 
-    test(`if specified it copies _version to version`, () => {
+    test(`if specified it encodes _seq_no and _primary_term to version`, () => {
       const serializer = new SavedObjectsSerializer(new SavedObjectsSchema());
       const actual = serializer.rawToSavedObject({
         _id: 'foo:bar',
-        _version: 4,
+        _seq_no: 4,
+        _primary_term: 1,
         _source: {
           type: 'foo',
           hello: {},
         },
       });
-      expect(actual).toHaveProperty('version', 4);
+      expect(actual).toHaveProperty('version', encodeVersion(4, 1));
```

Comment:
Since we rely on the result of `encodeVersion`, I would like to see some test to explicitly test that function.